src/main/kotlin/net/arcanumverum/arcanelibraries/items/scribingtools.kt

```diff
@@ -0,0 +1,9 @@
+package net.arcanumverum.arcanelibraries.items
+
+import net.minecraft.item.Item
+
+class ScribingToolsItem(settings: Settings) : Item(settings) {
+    init {
+        this.recipeRemainder = this
```

Comment:
This line gave me an error when I tried to build.
`Kotlin: Val cannot be reassigned`

Reply:
That should be handled by the access widener directives.  Try running the `genSources` gradle task.